x-pack/solutions/security/plugins/security_solution/public/onboarding/components/onboarding_body/cards/common/connectors/connector_cards.tsx

```diff
@@ -14,7 +14,6 @@ import { ConnectorsMissingPrivilegesCallOut } from './missing_privileges';
 import type { AIConnector } from './types';
 import { ConnectorSetup } from './connector_setup';
 import { ConnectorSelectorPanel } from './connector_selector_panel';
-import { AIActionTypeIds } from './constants';
 
 interface ConnectorCardsProps {
```

Comment:
I think this component can be moved to `security_solution/public/common` as well, since it has nothing to do with Onboarding as such.

I was actually starting to do that for `Reprocess Rules` functionality. See if you can do that or I am also okay doing it in follow up PR.

Reply:
That's a good idea, but it would be better to move it when we start importing it from outside the onboarding.